File: sdkwork-ai-sdk-flutter/lib/src/api/moderations2.dart
import '../http/client.dart';
import '../models.dart';

class Moderations2Api {
  final HttpClient _client;
  
  Moderations2Api(this._client);

  /// Create moderation
  Future<ModerationResponse?> createModeration(ModerationRequest body) async {
    final response = await _client.post(ApiPaths.aiPath('/moderations'), body: body, contentType: 'application/json');
    return response is ModerationResponse ? response : null;
  }
}

File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/api/moderations2.py
from typing import Any, Dict, List, Optional
from ..http_client import HttpClient
from ..models import ModerationRequest, ModerationResponse

class Moderations2Api:
    """moderations API client."""
    
    def __init__(self, client: HttpClient):
        self._client = client

    def create_moderation(self, body: ModerationRequest) -> ModerationResponse:
        """Create moderation"""
        return self._client.post(f"/ai/v3/moderations", json=body)
